README.md

```diff
@@ -1,6 +1,6 @@
 # bundlebun
 
-**bundlebun bundles [Bun](https://bun.sh), a fast JavaScript runtime, package manager, and builder, with your Ruby and Rails applications**. No need to use Docker, devcontainers, `curl ... | sh`, or `brew`.
+**bundlebun bundles [Bun](https://bun.sh) (a fast JavaScript runtime, package manager, and builder) with your Ruby and Rails applications**. No need to use Docker, devcontainers, `curl ... | sh`, or `brew`.
```

Comment:
```suggestion
**bundlebun bundles [Bun](https://bun.sh), a fast JavaScript runtime, package manager, and builder, with your Ruby and Rails applications**. No need to use Docker, devcontainers, `curl ... | sh`, or `brew`.
```
I get that it is stylistic, but rather leave as is. emdash would be perfect, but would break in some fields (I copy-paste the description)